### ISSUE TITLE
Pangram instructions are misleading

### ISSUE THREAD

**Issue by maddie927** [exercism/purescript#99]
> The instructions are misleading in claiming that "Input will not contain non-ASCII symbols"!

https://exercism.io/mentor/solutions/79d17dc77b484554b18fb1c6c5c5f015

**Comment by mawis** [comment]
I agree, that the `README.md` is not consistent with the tests in `Main.purs`.

We have two options to fix this:
* update the `README.md` or
* remove the test “pangram with non ascii characters” from `Main.purs`.

The [problem specification for the pangram exercise](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json) does not contain a test with non-ASCII characters. Actually I also don't see what is the purpose of having this test in the test suite. Is there any trivial solution to the exercise, that would pass the tests without this test, but fail on that test?

So I tend to remove the test case. What do you think?

**Comment by mawis** [comment]
While reading closed issues I found these as related: #53 #55

**Comment by BNAndras** [comment]
All tests are ASCII-only now
